src/regular-expression-comment.ts

```diff
@@ -24,10 +24,28 @@ export default {
           const regexLine = node.loc?.start.line;
           if (regexLine) {
```

Comment:
this would fail the new truthiness check we do as part of our normal config, could you add that to the rules for this repo and fix?